activmusic/apps/uploadmgr/templates/uploadmgr/index.html

```diff
@@ -8,24 +8,28 @@
         <a href="{% url 'music_add' %}" class="btn btn-primary">Add Another</a>
     </h1>
 
-    {% for media in medias %}
-    {% if forloop.first %}
-    <table class="table">
-        <tr>
-            <th>{% trans "Name" %}</th>
-            <th>{% trans "URL" %}</th>
-            <th class="table-col-delete">{% trans "Delete" %}</th>
-        </tr>
-    {% endif %}
-        <tr>
-            <td>{{ media.name|default:"[not downloaded yet]" }}</td>
-            <td><a href="{{ media.url }}" target="_blank">{{ media.url }}</a></td>
-            <td><input type="checkbox"/></td>
-        </tr>
-    {% if forloop.last %}
-    </table>
-    {% endif %}
-    {% empty %}
-        <p>{% trans "You don't have uploaded anything yet!" %}</p>
-    {% endfor %}
+    <form method="post" action="{% url 'music_delete' %}">
+        {% csrf_token %}
+        <button class="btn btn-danger">Remove all selected tracks</button>
+        {% for media in medias %}
+        {% if forloop.first %}
+        <table class="table">
```

Comment:
Why did you not start the form here and put the closing form tag below the {{ if forloop.last }} ? I mean, choosing this way, the form will not appear for people that not have uploaded music yet :)